xbmc/cores/VideoPlayer/test/edl/TestEdl.cpp

```diff
@@ -49,16 +49,16 @@ TEST_F(TestEdl, TestParsingMplayerTimeBasedEDL)
   // file has only 1 cut starting at 5.3 secs and ending at 7.1 secs
   // so total cut time should be 1.8 seconds (1800 msec)
   EXPECT_EQ(edl.HasCuts(), true);
-  EXPECT_EQ(edl.GetTotalCutTime(), 1.8 * 1000);
+  EXPECT_EQ(edl.GetTotalCutTime(), 1800ms);
```

Comment:
Should there be an include in this file now?

```
 #include <chrono>
```

Reply:
The Edit struct from the EDLEdit.h header is based on chrono and includes it, but I'll follow your suggestion in case it is somehow forward declared in the future